ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/site/js/formcontainerview.js

```diff
@@ -76,7 +78,11 @@
             let formContainer =  e.detail;
             let formEl = formContainer.getFormElement();
             setTimeout(() => {
-                formEl.classList.remove(FormContainerV2.loadingClass);
+                //for loop added to remove the applied loading class from the newly added Div in html
+                for (let i = 0; i < elements.length; i++) {
+                    elements[i].classList.remove(FormContainerV2.loadingClass);
```

Comment:
onInit method would be called each time for all the form in a page. removing the loading class in a form loop without depending on the form container element that is initialised looks incorrect.

Reply:
But the onInit is defined within OnDocument ready and while adding the Loading class we have used querySelectorAll() due to which we are removing it here in the For loop. On removing only one from the classList it's not getting removed